helm-charts/vmpooler/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: vmpooler
 description: A Helm chart to deploy vmpooler
 type: application
 icon: https://github.com/puppetlabs/vmpooler/raw/master/lib/vmpooler/public/img/logo.png
-version: 1.7.1
-appVersion: 1.0.0-prod-all-providers
+version: 1.10.0
```

Comment:
Why are you skipping 1.8 and 1.9?

Reply:
yeah so maybe I'm not following properly because I saw that we were using 1.10 and then reverted to 1.8 because of the ABS issue, the helm chart used in prod is 1.8 now but 1.10 exists too.

I just looked at the artifactory repo for helm-puppet and we have dio-vmpooler- up to 1.11.1 but maybe thats just the dio-helm-charts resulting charts and I got confused. 

this version value here, is it pushed somewhere and can I check the single source of truth to know what to use next?